packages/DatePicker/src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+import momentPropTypes from "react-moment-proptypes";
+
+import "react-dates/initialize";
+import { DayPickerSingleDateController as SDPController } from "react-dates";
+import "react-dates/lib/css/_datepicker.css";
```

Comment:
@nahumzs what do we do about the CSS dependency?

Reply:
If we want to get rid of any css files, I can move this 3rd party code into our `styles.js`

Reply:
we are not supporting it, let's move the css into a styles.js file. Let's keep the library 100% javascript friendly.